packages/@aws-cdk/aws-scheduler-alpha/jest.config.js

```diff
@@ -0,0 +1,10 @@
+const baseConfig = require('@aws-cdk/cdk-build-tools/config/jest.config');
+module.exports = {
+    ...baseConfig,
+    coverageThreshold: {
+        global: {
+            ...baseConfig.coverageThreshold.global,
+            branches: 60,
```

Comment:
Set coverage to 60, because current coverage (while sufficient IMO) is at 67%. We can remove this override later

Reply:
the tests seem pretty comprehensive, so im fine with this. if you look at the `coverage` folder of your local checkout, you should be able to see what branches are not being covered and determine if they are necessary for unit tests.